src/stcal/jump/twopoint_difference.py

```diff
@@ -138,7 +164,7 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
                                   max_ratio > two_diff_rej_thresh))
 
         log_str = 'From highest outlier, two-point found {} pixels with at least one CR from {} groups.'
-        log.info(log_str.format(len(row4cr), 'five'))
+        log.info(log_str.format(len(row4cr), 'five or more'))
```

Comment:
Thanks for adding that. That's been bugging me for a while now!

Reply:
Same!